### ISSUE TITLE
Prep project boards for Migration

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/devops-security#38]
### Overview
We need to prepare the Project Boards for the forced migration that GitHub is doing in August, so that it is already working well for the team by then.

### Action Items
- [x]  copy this template to a comment below
   ````
   ### Text from project board card, [Add name of column here]
   ```
   [INSERT MARKDOWN HERE]
   ```
   - [ ] Determine where this item goes on the wiki (page and section)
   - [ ] copy item to wiki
   - [ ] hide this comment when completed.
   ````
- [x] copy markdown from a card in the "Start Here" column of the Project Board into your new comment where it says `[INSERT MARKDOWN HERE]
- [x] delete card from project board.

### Resources/Instructions
Project board URL: old project board deleted
New project board: https://github.com/orgs/hackforla/projects/73/views/1

**Comment by ExperimentsInHonesty** [comment]
### Text from DevOps Security Project Management Board  project board card, Start Here column
```
This project is managed by the Hack for L.A. [DevOps](https://github.com/hackforla/ops/) Community of Practice
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from DevOps Security Project Management Board project board card, Ice Box Column
```
Icebox = issues identified that are going to get addressed, but are not being started yet because of dependencies
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from DevOps Security Project Management Board project board card, New Issue Review
```
All issues that are newly created or coming out of the icebox need to be reviewed by Product to make sure they are complete and prioritized
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### ext from DevOps Security Project Management Board project board card, Prioritized Backlog column
```
Prioritized Backlog = features that are ready to work and a priority. This column should be prioritized by most important first
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from DevOps Security Project Management Board project board card, In Progress column
```
In Progress = tasks that are actively being worked on by an individual team member and have been assigned to them in the issue
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from DevOps Security Project Management Board project board card, Links/Questions/Review column
```
Links/Questions/Review = Cards that have links that you need to get to often; issues that you are blocked on because you need an answer from another team member or other type of question/review
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
All the text on each of the cards has been moved or deemed unnecessary.  See hidden comments for details of each card.
